### DISCUSSION TITLE
Could the documentation add more content?

### DISCUSSION THREAD

**Discussion by MrLongBz** [reactiveui/ReactiveUI#3208] [Q&A]
Can WinFrom's event binding be multi-point examples? Can the content of the document be improved? Really need help with documentation? It would be even better if I could add Chinese. I am very grateful for this project's help to me.

**Comment by glennawatson** [comment]
This has been added to the wrong repository. Try https://github.com/reactiveui/website

**Comment by glennawatson** [comment]
To give a answer anyway. 

Chinese translation would only happen if we had volunteers to convert which is unlikely. 

WinForms Events would be the similar as other platforms so that could be a starting point. 

Documentation can always be improved and be happy to have the assistance. Join the slack channels if you are interested in helping.
